### PR TITLE
Pkcs11 misc

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: ta/pkcs11/src/pkcs11_token.c [hunk 1]
```diff
@@ -683,16 +683,16 @@ uint32_t entry_ck_close_all_sessions(struct pkcs11_client *client,
...
 {
 	const uint32_t exp_pt = TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INOUT,
 						TEE_PARAM_TYPE_NONE,
 						TEE_PARAM_TYPE_MEMREF_OUTPUT,
 						TEE_PARAM_TYPE_NONE);
-	TEE_Param *ctrl = &params[0];
-	TEE_Param *out = &params[2];
-	uint32_t rv = 0;
+	TEE_Param *ctrl = params;
+	TEE_Param *out = params + 2;
```
Comment (jforissier) [inline ta/pkcs11/src/pkcs11_token.c:694]: Belongs in commit "ta: pkcs11: replace complicated params pointer calculation"

**Comment by jenswi-linaro** [comment]
Commits fixed up and tags applied.

**Comment by Emantor** [comment]
Minor nit: double `with` in the commit message for "ta: pkcs11: replace complicated params pointer calculation".
`Reviewed-by: Rouven Czerwinski <r.czerwinski@pengutronix.de>`

**Comment by jenswi-linaro** [comment]
Commit message update, tag applied to all commits. I hope that was what you meant.

**Pull request merged**